<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFormSystemSet</class>
 <widget class="QWidget" name="MainFormSystemSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainFormSystemSet {
	border-image: url(:/qrc/res/main_bk.png)  50 00 0 0 repeat;
}
QLabel { font:bold 22px;	color: rgb(255, 255, 255);}
QRadioButton {font:bold 22px;	color: rgb(255, 255, 255); }
QGroupBox::title {
  subcontrol-origin: margin;
  subcontrol-position: top center;
  padding-top: 10px;  
  border-top: 1px solid rgb(255, 255, 255);
  font:bold 24pt;
  color: rgb(255, 255, 255);
}
QGroupBox {
font: 75 18px;
background-color: rgba(85, 85, 127, 50);
border-radius: 10px;
border: 1px solid rgb(106, 163, 193);
}
QLineEdit {
font: bold 32px; 
background-color: rgb(255, 255, 255);
border-radius: 4px;
border: 1px solid rgba(39, 22, 22, 254);
}
QPushButton {	
   border-radius:10px;
	background-color: rgb(57, 57, 118);
	color: rgb(255, 255, 255);	
	font: bold 33px;
	border-style:solid;
}
QPushButton:pressed{			
	background-color: rgb(57, 56, 76);
	color: rgb(255, 255, 255);		
	font: bold 33px;
	border-style:solid;
}
QPushButton:checked {
	background-color: rgb(31, 17, 108);
	color: rgb(255, 255, 255);		
	font: bold 33px;
	border-style:solid;
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>951</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 1px solid  rgb(0, 0, 0);
	border-radius: 0px;
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.505682 rgba(35, 97, 222, 247), stop:0.539773 rgba(61, 114, 223, 247), stop:1 rgba(93, 142, 239, 238));
	color: rgb(255, 255, 255);
	font: 16px;
}</string>
   </property>
   <property name="title">
    <string>连接配置</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_mqttUrl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>591</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器URL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_MySN">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mqttUser">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mqttPass">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_mqtt">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-left-radius:10px;
border-bottom-left-radius:10px;</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel_mqtt">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>230</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MqttStatusMsg">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>启动成功</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btExitApp">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>600</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>951</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>温度校准</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>原始值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>校准值</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_net_eth">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>610</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>有线网络设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_net_wlan">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>600</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>无线网络设置</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>951</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>外部输入来源</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>I/O 外部模块</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_KangUart">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> font:bold 22px;	color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SaveKangUart">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_StatusKang">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>启动成功</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
